packages/compass-crud/src/components/document-list.jsx

```diff
@@ -82,6 +92,30 @@ class DocumentList extends React.Component {
     );
   }
 
+  /**
+   * Render the fetching indicator with cancel button
+   */
+  renderFetching() {
+    return (
+      <div className="document-list-fetching">
```

Comment:
Didn't seem too trivial because compass-components would be where that goes and that uses different css and testing stuff. Not that this has tests or very much CSS. I guess I can do that..